WordPress/src/main/java/org/wordpress/android/ui/sitecreation/NewSiteCreationSiteInfoFragment.kt

```diff
@@ -104,6 +110,9 @@ class NewSiteCreationSiteInfoFragment : NewSiteCreationBaseFormFragment<NewSiteC
                 }
             }
         })
+        viewModel.onHelpClicked.observe(this, Observer {
+            helpClickedListener.onHelpClicked(HelpActivity.Origin.NEW_SITE_CREATION_SITE_INFO)
```

Comment:
I don't have a preference one way or another, but should the origin be coming from the VM instead?

Reply:
Even if that's a good idea, we probably shouldn't do it in this PR, because I think we'll need to move the `Origin` out of `HelpActivity`, because it's kind of weird to import something from the Activity in the VM.

Reply:
@oguzkocer That was exactly my reasoning why I didn't put it into the VM. But I agree it'd be cleaner.